initial_script.py

```diff
@@ -0,0 +1,80 @@
+# import
+import pandas as pd
+import numpy as np
+
+# load ratings
+ratings = pd.read_csv('Downloads/BX-Book-Ratings.csv', encoding='cp1251', sep=';')
+ratings = ratings[ratings['Book-Rating'] != 0]
+
+# load books
+books = pd.read_csv('Downloads/BX-Books.csv', encoding='cp1251', sep=';', error_bad_lines=False)
```

Comment:
Dataset je mozne zpracovat: jsou tady treba NaN hodnoty u sloupce Yeatr-of-Publication, Publisher a Book-Author, nebo sloupce s URLami obrazkami, ktere se nepouzivaji.